swagger.go

```diff
@@ -106,17 +109,26 @@ func Oauth2DefaultClientID(oauth2DefaultClientID string) func(*Config) {
 	}
 }
 
+// UsePkceWithAuthorizationCodeGrant enables Proof Key for Code Exchange.
+// Only applies to accessCode (Authorization Code) flows.
+func UsePkceWithAuthorizationCodeGrant(usePkce bool) func(*Config) {
```

Comment:
Please rename the option to `Oauth2UsePkce` and you can leave the details the function doc.